mods/cnc/maps/gdi10a/gdi10a.lua

```diff
@@ -0,0 +1,413 @@
+--[[
+   Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+Difficulty = Map.LobbyOption("difficulty")
+
+-- Destroy all to unlock airstrike
+SamSites = { sam1, sam2, sam3, sam4 }
+
+-- Buildings to rebuild if destroyed
+RebuildableStructs = { gun1, gun2, gun3, gun4, obelisk1, power1, power2, power3, power4,
+	power5, power6, power7, power8, airfield, hand, refinery }
+
+-- Attack waves
+AttackPaths =
+{
+	{ waypoint11.Location, waypoint18.Location, waypoint12.Location, waypoint3.Location, waypoint4.Location, waypoint5.Location, waypoint6.Location },
+	{ waypoint11.Location, waypoint18.Location, waypoint12.Location, waypoint0.Location, waypoint2.Location, waypoint13.Location, waypoint14.Location, waypoint15.Location, waypoint17.Location },
+	{ waypoint11.Location, waypoint18.Location, waypoint12.Location, waypoint0.Location, waypoint7.Location, waypoint8.Location, waypoint9.Location, waypoint10.Location, waypoint17.Location }
+}
+-- Time between attack waves
+AttackCooldown =
+{
+	hard = DateTime.Seconds(10), normal = DateTime.Seconds(20), easy = DateTime.Seconds(30)
+}
+-- Armor/vehicle production
+ArmorProducer = {}
+ArmorProducer.Cooldown = AttackCooldown[Difficulty]
+-- kind of mimics what the original mission sends at you
+ArmorProducer.ModelGroups = {
+	{ "arty", "arty" },
+	{ "bggy", "ltnk" },
+	{ "ltnk", "ltnk" },
+}
+ArmorProducer.AttackPaths = AttackPaths
+ArmorProducer.AttackGrp = {}
+
+-- Infantry production
+InfProducer = {}
+InfProducer.Cooldown = AttackCooldown[Difficulty]
+-- kind of mimics what the original mission sends at you
+InfProducer.ModelGroups = {
+	{ "e1", "e1", "e3", "e3", "e4", "e4" },
+	{ "e3", "e3", "e4", "e4" },
+}
+InfProducer.AttackPaths = AttackPaths
+InfProducer.AttackGrp = {}
+
+-- Build queue for structures
+CyardBuildQueue = {}
+
+WorldLoaded = function()
+	print('====  WorldLoaded  ====')
+	GDI = Player.GetPlayer("GDI")
+	Nod = Player.GetPlayer("Nod")
+
+	-- Prepare Objectives
+	KillNod = GDI.AddObjective("Destroy all Nod units and buildings.")
+	KillGDI = Nod.AddPrimaryObjective("Kill GDI")
+
+	AirSupport = GDI.AddObjective("Destroy the SAM sites to receive air support.", "Secondary", false)
+	Trigger.OnAllKilled(SamSites, function()
+		GDI.MarkCompletedObjective(AirSupport)
+		Actor.Create("airstrike.proxy", true, { Owner = GDI })
+	end)
+
+	InitObjectives(GDI)
+
+	--TODO DefaultCameraPosition is too far west??
```

Comment:
Your confusion is understandable. In TD '95, that waypoint is intended to be the upper-left corner of the starting camera. In OpenRA, a target is chosen as the camera's center.

Reply:
Since I couldn't identify which waypoint to use, I've centred the camera on the MCV during map start.  I will remove the old, commented-out lines...